bt-dht/the8472/mldht/src/lbms/plugins/mldht/DHTConfiguration.java

```diff
@@ -16,28 +16,28 @@
 	/**
```

Comment:
the8472/mldht source root should not be modified, as it is merged and updated (manually) from upstream repo